omod/src/main/webapp/resources/scripts/services/privilegeService.js

```diff
@@ -0,0 +1,45 @@
+angular.module('privilegeService', ['ngResource'])
```

Comment:
This is what i said, see how this change is creeping into this pull request? You should have first reverted this commit before creating this branch